### PR TITLE
feat(navbar): Added a link to project support in help center

### PR DESCRIPTION
**Pull request by qbzzt**
https://help.optimism.io/hc/en-us/articles/9762044018843-How-do-I-get-project-support-marketing-integrations-etc-

### CODE REVIEW

**Comment by Nicca42** [comment]
@qbzzt I wanted this info to be native to the docs not just in the user docs.

**Comment by qbzzt** [comment]
> @qbzzt I wanted this info to be native to the docs not just in the user docs.

I could copy it over. But what would be the benefit? By having it in one place there is just one place to update when things change.

**Comment by Nicca42** [comment]
Yea this should be native in the docs.

**Comment by qbzzt** [comment]
> Yea this should be native in the docs.

With @zainbacchus 's approval, I moved his stuff to a markdown. The style is very different from the one I use. @Nicca42 , your call if it's appropriate here or if I should rewrite it.

File: src/docs/biz/README.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+1. See our [Developer Documentation](../developers/), [Tutorials](https://github.com/ethereum-optimism/optimism-tutorial), and [Help Center](https://help.optimism.io)
+1. Head over to [**#dev-support**](https://discord.com/channels/667044843901681675/887914409207414785) on discord for the fastest help 
+1. Join [this TG group](https://t.me/+Cb7q0a1YqItkZTRk) and ask for help there
+1. Last attempt: [Fill out this form](https://optimismpbc.typeform.com/get-in-touch?typeform-source=community.optimism.io)
+
+
+## Marketing Requests 🦸🦸
+
+When your project is deployed on Optimism, you’ll be added to [our ecosystem page](https://www.optimism.io/apps/all).
```
Comment (Nicca42) [inline src/docs/biz/README.md:30]:
```suggestion
When your project is deployed on Optimism, you can be added to [our ecosystem page](https://www.optimism.io/apps/all). 
```

File: src/docs/biz/README.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+1. See our [Developer Documentation](../developers/), [Tutorials](https://github.com/ethereum-optimism/optimism-tutorial), and [Help Center](https://help.optimism.io)
+1. Head over to [**#dev-support**](https://discord.com/channels/667044843901681675/887914409207414785) on discord for the fastest help 
+1. Join [this TG group](https://t.me/+Cb7q0a1YqItkZTRk) and ask for help there
+1. Last attempt: [Fill out this form](https://optimismpbc.typeform.com/get-in-touch?typeform-source=community.optimism.io)
+
+
+## Marketing Requests 🦸🦸
+
+When your project is deployed on Optimism, you’ll be added to [our ecosystem page](https://www.optimism.io/apps/all). 
+Simply [fill out this form](https://optimismpbc.typeform.com/op-marketing) to be included.
```
Comment (Nicca42) [inline src/docs/biz/README.md:31]:
```suggestion
Simply [fill out this form](https://optimismpbc.typeform.com/op-marketing) to be included. Inclusion is at our discretion. 
```

File: src/docs/biz/README.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+1. Head over to [**#dev-support**](https://discord.com/channels/667044843901681675/887914409207414785) on discord for the fastest help 
+1. Join [this TG group](https://t.me/+Cb7q0a1YqItkZTRk) and ask for help there
+1. Last attempt: [Fill out this form](https://optimismpbc.typeform.com/get-in-touch?typeform-source=community.optimism.io)
+
+
+## Marketing Requests 🦸🦸
+
+When your project is deployed on Optimism, you’ll be added to [our ecosystem page](https://www.optimism.io/apps/all). 
+Simply [fill out this form](https://optimismpbc.typeform.com/op-marketing) to be included. 
+We add new projects [here](https://www.optimism.io/apps/all) every week, but you need to deploy first.
```
Comment (Nicca42) [inline src/docs/biz/README.md:32]:
```suggestion
We add new projects live on the Optimism mainnet [here](https://www.optimism.io/apps/all) once a week.
```

File: src/docs/biz/README.md [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+## Marketing Requests 🦸🦸
+
+When your project is deployed on Optimism, you’ll be added to [our ecosystem page](https://www.optimism.io/apps/all). 
+Simply [fill out this form](https://optimismpbc.typeform.com/op-marketing) to be included. 
+We add new projects [here](https://www.optimism.io/apps/all) every week, but you need to deploy first. 
+
+
+### Promoting Values before anything else
+
+We love to promote values-aligned projects as part of our broader content program.
```
Comment (Nicca42) [inline src/docs/biz/README.md:37]: Do we have our values anywhere publicly?

File: src/docs/biz/README.md [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+Other than RT's, we may like or reply to a tweet or include it in our weekly wrap up thread. 
+You're welcome to submit your tweet for amplification but please keep in mind if and how we interact with it is at our sole discretion.
+
+
+### OPRadio
+
+We also host a weekly Twitter Space in an interview/podcast format called OP Radio. You're welcome to express interest in being featured on OP Radio, but keep in mind that we maintain the final say on whether it makes sense or not.
+
+ 
+## Grants Funding 🪙🪙
```
Comment (Nicca42) [inline src/docs/biz/README.md:52]:
```suggestion
## Governance Grants
```

File: src/docs/biz/README.md [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+
+Grants are mainly handled by [The Optimism Collective](https://app.optimism.io/announcement), our open governance system/digital society. If you are interested in seeking funding here, we first recommend the following:
```
Comment (Nicca42) [inline src/docs/biz/README.md:54]:
```suggestion
Grants are mainly handled by [The Optimism Collective](https://app.optimism.io/announcement), our open governance system/digital society. If you are interested in applying, we first recommend the following: 
```

File: src/docs/biz/README.md [hunk 7]
```diff
@@ -0,0 +1,109 @@
...
+
+We also host a weekly Twitter Space in an interview/podcast format called OP Radio. You're welcome to express interest in being featured on OP Radio, but keep in mind that we maintain the final say on whether it makes sense or not.
+
+ 
+## Grants Funding 🪙🪙
+
+Grants are mainly handled by [The Optimism Collective](https://app.optimism.io/announcement), our open governance system/digital society. If you are interested in seeking funding here, we first recommend the following: 
+
+::: tip Tip
+Grants are a lever to support sustainable growth in the ecosystem, therefore we do not provide grants for projects' operational expenses.
```
Comment (Nicca42) [inline src/docs/biz/README.md:57]: Remove this, we are doing something along these lines for season 3. Instead:

Before applying, please check that your application falls within the possible Governance Grant categories in the [OPerating manual](https://github.com/ethereum-optimism/OPerating-manual/blob/main/manual.md#valid-proposal-types).

File: src/docs/biz/README.md [hunk 8]
```diff
@@ -0,0 +1,109 @@
...
+
+Grants are mainly handled by [The Optimism Collective](https://app.optimism.io/announcement), our open governance system/digital society. If you are interested in seeking funding here, we first recommend the following: 
+
+::: tip Tip
+Grants are a lever to support sustainable growth in the ecosystem, therefore we do not provide grants for projects' operational expenses. 
+:::
+ 
+
+- Deploy to Optimism (we are [EVM Equivalent](https://medium.com/ethereum-optimism/introducing-evm-equivalence-5c2021deb306), not just EVM compatible so it is very easy to deploy). 
+  Check out [our tutorial](https://github.com/ethereum-optimism/optimism-tutorial/tree/main/getting-started) to see how to deploy.
```
Comment (Nicca42) [inline src/docs/biz/README.md:62]:
```suggestion
  Check out our [getting started tutorial](https://github.com/ethereum-optimism/optimism-tutorial/tree/main/getting-started) to see how to deploy. We also have [tons of more in-depth tutorials](tutorials) for bridging and other L2 specific functionality. 
```

File: src/docs/biz/README.md [hunk 9]
```diff
@@ -0,0 +1,109 @@
...
+- Try gaining organic traction:
+  - Traction can be gained through our community.
+  - Head over to [our discord](https://discord-gateway.optimism.io/) and check out [#showandtell](https://discord.com/channels/667044843901681675/819333988719263774) or [#demoday](https://discord.com/channels/667044843901681675/960493820674465883).
+  Demoday is an AMA about your product to our community.
```
Comment (Nicca42) [inline src/docs/biz/README.md:66]:
```suggestion
  Demo Day is an AMA about your product to our community. Apply in Discord. 
```

File: src/docs/biz/README.md [hunk 10]
```diff
@@ -0,0 +1,109 @@
...
+  - Seek synergies – look at other teams already deployed on Optimism that could work with your product and reach out to them.
+- Once you have done the above, check out our [governance grant process](https://gov.optimism.io/t/grant-proposal-template/3233).
+- To get reviews in real time from governance delegates go to discord and post about your product in [#temp-check](https://discord.com/channels/667044843901681675/1011238484373159956).
```
Comment (Nicca42) [inline src/docs/biz/README.md:69]:
```suggestion
- To get reviews in real time from governance delegates go to discord and post about your project in [#temp-check](https://discord.com/channels/667044843901681675/1011238484373159956).
```

File: src/docs/biz/README.md [hunk 11]
```diff
@@ -0,0 +1,109 @@
...
+
+## Other Helpful Links
+
+Curious about our Optimistic Origins? 
+Listen to [this OPRadio Episode](https://twitter.com/optimismFND/status/1585015947626426368) with one of our founders, [@JingleJam](https://twitter.com/jinglejamOP).
+
+What do the next 100 years have in store for crypto? [@Karl_dot_tech](https://twitter.com/karl_dot_tech) shares his vision in [this chat with Bankless](https://www.youtube.com/watch?v=60F2wtjZO5M).
+
+Want to see what we are doing with Retroactive Public Goods? 
+(Hint: We give all our sequencer ~100k/day in revenue, away to public goods so your users can rest easy knowing that when they use Optimism, they are contributing to a good cause).
```
Comment (Nicca42) [inline src/docs/biz/README.md:80]:
```suggestion
(Hint: We give all our sequencer revenue (~100k/day!), away to public goods so your users can rest easy knowing that when they use Optimism, they are contributing to a good cause). 
```

File: src/docs/biz/README.md [hunk 12]
```diff
@@ -0,0 +1,109 @@
...
+## Other Helpful Links
+
+Curious about our Optimistic Origins? 
+Listen to [this OPRadio Episode](https://twitter.com/optimismFND/status/1585015947626426368) with one of our founders, [@JingleJam](https://twitter.com/jinglejamOP).
+
+What do the next 100 years have in store for crypto? [@Karl_dot_tech](https://twitter.com/karl_dot_tech) shares his vision in [this chat with Bankless](https://www.youtube.com/watch?v=60F2wtjZO5M).
+
+Want to see what we are doing with Retroactive Public Goods? 
+(Hint: We give all our sequencer ~100k/day in revenue, away to public goods so your users can rest easy knowing that when they use Optimism, they are contributing to a good cause). 
+More on Public Goods Funding [here](https://app.optimism.io/announcement), [here](https://medium.com/ethereum-optimism/retroactive-public-goods-funding-33c9b7d00f0c), and [Vitalik’s](https://twitter.com/VitalikButerin) review of our first experimental round is [here](https://vitalik.ca/general/2021/11/16/retro1.html).
```
Comment (Nicca42) [inline src/docs/biz/README.md:81]: Can we move this to its own page? Then add little one liners about each of these. We really need to have PGF as its own page under the citizen's house.

**Review by Nicca42 — CHANGES_REQUESTED** [review]

**Comment by Nicca42** [comment]
I'd like to get this in soon ish, we are going to have more eyes looking for support with the retroPGF hype.

File: src/docs/biz/README.md (hunk 12 above)
Comment (qbzzt) [reply src/docs/biz/README.md:77]: Now that there's a page for it (https://community.optimism.io/docs/governance/retropgf-2/), can we just remove the material from here and link to it?

Comment (qbzzt) [reply src/docs/biz/README.md]: If that is acceptable, just approve and merge - I changed it to that link and dealt with everything else.

**Comment by qbzzt** [comment]
Now that there's a RetroPGF-2 page, I just linked to that.

**Comment by qbzzt** [comment]
> Why is this PR now modifying all these other files? This PR should not include attestation station stuff. Can we please clean this up or check it is 💯 the latest.

Arrggghhh!!!! It now has over 150 commits. I'm going to close this PR and open a clean one.

**Pull request closed without merging**